mesonbuild/backend/ninjabackend.py

```diff
@@ -114,21 +114,25 @@ def get_rsp_threshold():
 # from, etc.), so it must not be shell quoted.
 raw_names = {'DEPFILE_UNQUOTED', 'DESC', 'pool', 'description', 'targetdep'}
 
+NINJA_QUOTE_BUILD = re.compile("[$ :\n]")
+NINJA_QUOTE_VAR = re.compile("[$ \n]")
```

Comment:
```suggestion
NINJA_QUOTE_BUILD = re.compile(r"[$ :\n]")
NINJA_QUOTE_VAR = re.compile(r"[$ \n]")
```

Values passed to `re` methods as regular expressions should be "raw" strings.